digits/templates/models/images/classification/new.html

```diff
@@ -376,14 +382,16 @@ <h4 style="display:inline-block;">Python Layers</h4>
               )
         .done(function(data) {
                 data = $.parseJSON(data);
-                $('textarea[name=custom_network]').val(data['network']);
+                editor.setValue(data['network']);
+                editor.setValue(editor.getValue(), 1); // Moves cursor to end of editor
```

Comment:
Opening a text document at the tail is nonstandard. Selecting all the text is really nonstandard. I suggest having it not select any text, and open at the head for this PR, and if you prefer, change it to open at the tail in your repo.